resource/js/components/SearchPage/SearchResultList.js

```diff
@@ -14,7 +14,12 @@ export default class SearchResultList extends React.Component {
     let returnBody = body;
 
     this.props.searchingKeyword.split(' ').forEach((keyword) => {
-      const k = keyword.replace(/[.*+?^${}()|[\]\\]/g, "\\$&");
+      if (keyword === '') {
+        return;
+      }
```

Comment:
Small fix. 
When the end of search keywords is `<SPACE>`, `returnBody` got unnecessary empty lines.